### PR TITLE
validation: add standard multisig addresses for guardian and challenger

### PR DESCRIPTION
**Pull request by geoknee**
Towards https://github.com/ethereum-optimism/client-pod/issues/913

- [x] Note that many chains currently fail this check, so will need to be ~bumped down to "frontier".~ excluded for now

### CODE REVIEW

**Comment by geoknee** [comment]
When https://github.com/ethereum-optimism/superchain-registry/pull/279 is merged, we should pull the key handover addresses into these config toml files (the test code can remain separate.

File: validation/standard/init.go [hunk 1]
```diff
@@ -7,16 +7,35 @@ import (
...
+	}
+
+	Config.MultisigRoles["sepolia"] = new(MultisigRoles)
+	err = decodeTOMLFileIntoConfig("standard-config-roles-sepolia.toml", Config.MultisigRoles["sepolia"])
+	if err != nil {
+		panic(err)
+	}
+
+	Config.MultisigRoles["sepolia-dev-0"] = new(MultisigRoles)
+	err = decodeTOMLFileIntoConfig("standard-config-roles-sepolia-dev-0.toml", Config.MultisigRoles["sepolia-dev-0"])
```
Comment (bitwiseguy) [inline validation/standard/init.go:38]: To further cut down on redundant code related to `MultisigRoles` and `Params` you could loop through all network names like this:
```
	networks := []string{"mainnet", "sepolia", "sepolia-dev-0"}
	for _, network := range networks {
		Config.MultisigRoles[network] = new(MultisigRoles)
		err = decodeTOMLFileIntoConfig("standard-config-roles-"+network+".toml", Config.MultisigRoles[network])
		if err != nil {
			panic(err)
		}

		Config.Params[network] = new(Params)
		err = decodeTOMLFileIntoConfig("standard-config-params-"+network+".toml", Config.Params[network])
		if err != nil {
			panic(err)
		}
	}
```
And perhaps move the `panic` inside `decodeTOMLFileIntoConfig` to reduce lines even more

**Review by bitwiseguy — APPROVED** [review]
Left one comment for potential optimization. Other than that LGTM!

File: validation/standard/init.go (hunk 1 above)
Comment (geoknee) [reply validation/standard/init.go]: [0a79b06](https://github.com/ethereum-optimism/superchain-registry/pull/300/commits/0a79b0608706551dee01073f16580ad75aaddf34)

**Pull request merged**